priv/static/index.html

```diff
@@ -30,51 +30,66 @@
     <nav>
       <ul>
         <li class="navbar"><a href="/index.html">tournamap.gg</a></li>
-        <li class="navbar" id="about-btn">About</li>
+        <li class="navbar" id="about-btn"><a href="#">About</a></li>
       </ul>
     </nav>
     <div id="main">
       <section id="about" hidden>
```

Comment:
When PR prompted me to read more about section elements, I ended up stumbling into `article` tags. If we're converting this into using section divs, do you mind converting this parent "about" section to use the article tag while you're at it?

Reply:
Article like section is a containing element, however, I would recommend not using an article element for this type of content. It is typically reserved for independently distributed content, such as a news article. Although, if you feel that this makes the most sense, I am more than happy to follow suit.

Reply:
I see. Thanks for the info, let's do without `article` then!